src/packages/console/src/pages/verifyEmailPage.tsx

```diff
@@ -46,32 +31,39 @@ export const VerifyEmailPage = (): React.ReactElement => {
   });
 
   return (
-    <Container component='main' maxWidth='sm'>
-      <Paper className={classes.paper}>
-        <Typography component='h1' variant='h5'>
+    <ContainingView isCenteredHorizontally>
+      <ResponsiveContainingView size={12} sizeResponsive={{ small: 8, medium: 6, large: 5 }}>
+        <Stack direction={Direction.Vertical} isFullHeight={true}>
+          <Stack.Item growthFactor={1} shrinkFactor={1} />
+          <Box variant='card' isFullWidth={false}>
+            <Text variant='header5' tag='h1' alignment={TextAlignment.Center}>
           Please check your email
-        </Typography>
-        <br />
-        <br />
-        <Typography>
+            </Text>
+            <br />
```

Comment:
use spacing instead of br now

Reply:
ok

Reply:
what variant should I use
to give space equivalent to 2 br's

Reply:
not sure, will have to see visually. I would guess `Wide` or `Wide2`. it doesnt have to match exactly what it was before, just needs to look good :)